lib/ClangImporter/ClangImporter.cpp

```diff
@@ -68,7 +68,9 @@ using namespace importer;
 using clang::CompilerInstance;
 using clang::CompilerInvocation;
 
+#if !defined(_MSC_VER)
 #pragma mark Internal data structures
+#endif
```

Comment:
You're right. I've updated #5904 to silence the warning, and updated this PR to remove all of these ifdefs